packages/web-frontend/src/components/Marker/markerColors.js

```diff
@@ -65,6 +65,7 @@ export function resolveSpectrumColour(scaleType, scaleColorScheme, value, min, m
 }
 
 const normaliseToPercentage = (value, min = 0, max = 1) => {
+  if (min === max) return 0.5;
```

Comment:
I would have thought 
```suggestion
  if (min === max) return 1;
```
was more appropriate? If you graphed the min tending towards the max, the percentage would get closer and closer to 1, and I wouldn't expect it to then jump back to 0.5. You may have a good reason for 0.5 though!

Reply:
> If you graphed the min tending towards the max, the percentage would get higher and higher and probably be at least 1

I'm not sure what you mean by this? To my mind, as the min tends to the max the percentage will get lower and lower (as it is the relative position of the value that we are interested in)

![image](https://user-images.githubusercontent.com/32168886/96196294-c639f780-0f9a-11eb-9210-6c844ee8558c.png)

There is however, an inconsistency as above, so we can do:
```
if value < min: 0
if value > max: 1
if min = max = value: 0.5
```
(but that isn't a use case at the moment, because of how scales are calculated)

Reply:
We know the value can't be outside the bounds of the min and max (that's kinda the definition, right?), so as the interval gets smaller, the percentage of that interval any number inside takes up becomes larger.

Reply:
The value can be outside the bounds, and is in some current use cases (covidau).

Also, I think it generally depends on if it is the min or the max that is changing. If it is the min that increases, the result will decrease, no?
![image](https://user-images.githubusercontent.com/32168886/96197950-216de900-0f9f-11eb-9a35-67a393c888c5.png)
![image](https://user-images.githubusercontent.com/32168886/96197942-187d1780-0f9f-11eb-8660-855fb2c22458.png)

Reply:
Oh really?? I thought the min and max were derived from the actual values. 

And yes, true about the min increasing.

Where do the min and max come from?

Reply:
`getSpectrumScaleValues` in `src/utils/measures`. There is various logic there to modify the min and max (although you are right, the default is to derive them from the actual values).

Reply:
Right! Well, your suggestion sounds great then:
if value < min: 0
if value > max: 1
if min = max = value: 0.5

Reply:
Sounds good, thanks Edwin 👍